plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,32 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getFullId() {
```

Comment:
I removed that one completely. Will go with another util method named lrtrim where we can trim a string with another one. In this case I compare the two strings and trim the "/" string.